dapr-spring-core/src/main/java/io/diagrid/spring/core/keyvalue/DaprKeyValueAdapter.java

```diff
@@ -112,27 +135,27 @@ public void clear() {
     public <T> Iterable<T> find(KeyValueQuery<?> query, String keyspace, Class<T> type) {
         DaprMetadata metadata = daprClient.getMetadata().block();
         List<ComponentMetadata> components = metadata.getComponents();
-        String statestoreTypeAndVersion = findStatestoreTypeAndVersion(statestoreName, components);
+        String stateStoreTypeAndVersion = findStateStoreTypeAndVersion(stateStoreName, components);
 
-        if (statestoreTypeAndVersion.isEmpty()) {
+        if (stateStoreTypeAndVersion.isEmpty()) {
             return Collections.emptyList();
         }
 
-        if (!statestoreTypeAndVersion.equals("state.postgresql-v1")) {
+        if (!stateStoreTypeAndVersion.equals("state.postgresql-v1")) {
             return Collections.emptyList();
         }
 
         SpelExpression expression = PARSER.parseRaw(query.getCriteria().toString());
         SpelNode left = expression.getAST().getChild(0);
         SpelNode right = expression.getAST().getChild(1);
 
-        String sqlText = "select regexp_replace(value#>>'{}', '\"', '\"', 'g') as value from state where key LIKE '" + statestoreName + "||" + type.getName() + "-%' and value->>"+left+"="+right;
+        String sqlText = "select regexp_replace(value#>>'{}', '\"', '\"', 'g') as value from state where key LIKE '" + stateStoreName + "||" + keyspace + "-%' and value->>"+left+"="+right;
```

Comment:
True ... I have left as is for now to focus on other operation like `clear`, `deleteAll`, `entries`, etc. Once those are done I want to focus on `find` separately since it is the most "sophisticated" one